frontend/src/lib/components/ActionSelectBox.js

```diff
@@ -1,16 +1,33 @@
-import React, { Component } from 'react'
+import React, { Component, useState } from 'react'
 import Select, { components } from 'react-select'
 import { ActionSelectInfo } from '../../scenes/trends/ActionSelectInfo'
 import { selectStyle } from '../utils'
 import PropTypes from 'prop-types'
+import ActionSelectTab from './ActionSelectTab'
 
-export class ActionSelectBox extends Component {
-    constructor(props) {
-        super(props)
-        this.state = {}
-    }
-    actionContains(action, event) {
-        return action.steps.filter(step => step.event == event).length > 0
+export function ActionSelectTabs(props) {
```

Comment:
should be quite reusable now

Reply:
I'm not sure why but clicking anywhere _but_ the select box doesn't make it go away.

Reply:
- [ ] It always opens the action tab, rather than the correct tab based on what's currently selected.

- [ ] The `Edit` link is always empty "", and the Select box is never automatically set to what action (or event) is curently selected.
![image](https://user-images.githubusercontent.com/1727427/78463293-e8d98280-76d2-11ea-85ee-34b3eb1010fe.png)